R/deployApp.R

```diff
@@ -365,6 +377,11 @@ deployApp <- function(appDir = getwd(),
     )
     taskComplete(quiet, "Visibility updated")
   }
+  if (length(target$envVars) > 0) {
+    taskStart(quiet, "Updating environment variables {envVars}...")
+    client$setEnvVars(application$guid, target$envVars)
```

Comment:
Document that environment variables are applied _BEFORE_ the deployment. Should the deploy fail, the env var updates will still be in effect.